app/Console/Commands/AdsharesCrawlCommand.php

```diff
@@ -89,14 +97,14 @@ protected function readBroadcasts(Esc $esc)
 
         if ($this->broadcast) {
             $this->info("Broadcast own host: $this->host");
-            $x = $esc->sendBroadcast($this->host);
+            $x = $adsClient->broadcast($this->host);
             var_dump($x);
             // TODO: this fails currently
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1135)